src/testRunner/unittests/incrementalParser.ts

```diff
@@ -832,157 +832,162 @@ module m3 { }\
+                    const index = getIndexOfTsIgnoreComment(atIndex);
+                    const source = textWithIgnoreCommentFrom(textWithIgnoreComment.slice(0, index) + textWithIgnoreComment.slice(index + tsIgnoreComment.length), singleIgnore);
+                    const oldText = ScriptSnapshot.fromString(source);
+                    const newTextAndChange = withInsert(oldText, index, tsIgnoreComment);
+                    verifyCommentDirectives(oldText, newTextAndChange);
+                }
+
+                function verifyChangeToBlah(atIndex: number, singleIgnore?: true) {
+                    const index = getIndexOfTsIgnoreComment(atIndex) + "// ".length;
+                    const oldText = ScriptSnapshot.fromString(textWithIgnoreCommentFrom(textWithIgnoreComment, singleIgnore));
+                    const newTextAndChange = withChange(oldText, index, 1, "blah ");
+                    verifyCommentDirectives(oldText, newTextAndChange);
+                }
+
+                function verifyChangeBackToDirective(atIndex: number, singleIgnore?: true) {
+                    const index = getIndexOfTsIgnoreComment(atIndex) + "// ".length;
+                    const source = textWithIgnoreCommentFrom(textWithIgnoreComment.slice(0, index) + "blah " + textWithIgnoreComment.slice(index + 1), singleIgnore);
+                    const oldText = ScriptSnapshot.fromString(source);
+                    const newTextAndChange = withChange(oldText, index, "blah ".length, "@");
+                    verifyCommentDirectives(oldText, newTextAndChange);
+                }
+
+                function verifyDeletingBlah(atIndex: number, singleIgnore?: true) {
+                    const tsIgnoreIndex = getIndexOfTsIgnoreComment(atIndex);
+                    const index = tsIgnoreIndex + "// ".length;
+                    const source = textWithIgnoreCommentFrom(textWithIgnoreComment.slice(0, index) + "blah " + textWithIgnoreComment.slice(index + 1), singleIgnore);
+                    const oldText = ScriptSnapshot.fromString(source);
+                    const newTextAndChange = withDelete(oldText, tsIgnoreIndex, tsIgnoreComment.length + "blah".length);
+                    verifyCommentDirectives(oldText, newTextAndChange);
+                }
+
+                function verifyChangeDirectiveType(atIndex: number, singleIgnore?: true) {
+                    const index = getIndexOfTsIgnoreComment(atIndex) + "// @ts-".length;
+                    const oldText = ScriptSnapshot.fromString(textWithIgnoreCommentFrom(textWithIgnoreComment, singleIgnore));
+                    const newTextAndChange = withChange(oldText, index, "ignore".length, "expect-error");
+                    verifyCommentDirectives(oldText, newTextAndChange);
+                }
+
+                function verifyReuseChange(atIndex: number, singleIgnore?: true) {
+                    const source = `const x = 10;
+    function foo1() {
+        const x1 = 10;
+        // @ts-ignore

```

Comment:
I am a little confused as to why these still pass without replacing `// @ts-ignore` with `${tsIgnoreComment}`... leaving as-is so my confusion is more visible.